curvefs/src/metaserver/copyset/conf_epoch_file.h

```diff
@@ -0,0 +1,80 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Created Date: 18-12-20
+ * Author: wudemiao
+ */
+
+#ifndef CURVEFS_SRC_METASERVER_COPYSET_CONF_EPOCH_FILE_H_
+#define CURVEFS_SRC_METASERVER_COPYSET_CONF_EPOCH_FILE_H_
+
+#include <memory>
+#include <string>
+
+#include "curvefs/proto/copyset.pb.h"
+#include "curvefs/src/metaserver/common/types.h"
+#include "src/fs/local_filesystem.h"
+
+namespace curvefs {
+namespace metaserver {
+namespace copyset {
+
+// TODO(wuhanqing): this file is identical to src/chunkserver/conf_epoch_file.h
+
+class ConfEpochFile {
+ public:
+    explicit ConfEpochFile(curve::fs::LocalFileSystem* fs) : fs_(fs) {}
+
+    /**
+     * 加载快照文件中的配置版本
```

Comment:
english

Reply:
I will fix this later with TODO in line 37